src/nvim/api/autocmd.c

```diff
@@ -197,6 +197,8 @@ Array nvim_get_autocmds(Dict(get_autocmds) *opts, Error *err)
             "pattern",
             STRING_OBJ(cstr_to_string((char *)ap->pat)));
 
+        PUT(autocmd_info, "event", INTEGER_OBJ(event));
```

Comment:
any reason we cannot use `event_nr2name` ? (afaik, event id is not something we expose, so the number does not mean lot to plugins). It was mentioned in chat but I didn't get the conclusion :)

Reply:
I would like to know that reason myself ;)

I just get an error about implicit declaration when I try to use it.

Reply:
> I just get an error about implicit declaration when I try to use it.

It is declared `static` in `nvim/autocmd.c`, if you remove the `static` keyword it should "just work".